galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.mq.manager/bnd.bnd

```diff
@@ -0,0 +1,22 @@
+-snapshot: ${tstamp}
+Bundle-Name: MQ Manager
+Export-Package: !dev.galasa.mq.internal*,\
+        dev.galasa.mq*,\
+        javax.jms*,\
+        javax.management*,\
+        com.ibm.msg.client.wmq*
```

Comment:
can't see any tpi/spi that uses com.ibm.msg.client.wmg* package,  so should not export them.

Reply:
done - updated